frontend/pages/browse.js

```diff
@@ -1,21 +1,25 @@
-import useScrollTrigger from "@material-ui/core/useScrollTrigger";
+// 3rd party or built-in imports
+import _ from "lodash";
 import axios from "axios";
 import NextCookies from "next-cookies";
 import React, { useRef, useState } from "react";
-import tokenConfig from "../public/config/tokenConfig";
-import { buildUrlEndingFromFilters } from "../public/lib/filterOperations";
+import useScrollTrigger from "@material-ui/core/useScrollTrigger";
+
+// Relative imports
 import {
   getOrganizationTagsOptions,
   getProjectTagsOptions,
   getSkillsOptions,
   getStatusOptions,
   membersWithAdditionalInfo,
 } from "../public/lib/getOptions";
+import { encodeQueryParamsFromFilters } from "../public/lib/urlOperations";
 import { parseData } from "../public/lib/parsingOperations";
 import BrowseContent from "../src/components/browse/BrowseContent";
-import TopOfPage from "../src/components/hooks/TopOfPage";
 import HubsSubHeader from "../src/components/indexPage/HubsSubHeader";
 import MainHeadingContainerMobile from "../src/components/indexPage/MainHeadingContainerMobile";
+import tokenConfig from "../public/config/tokenConfig";
+import TopOfPage from "../src/components/hooks/TopOfPage";
```

Comment:
Alphabetized imports, and grouped into 3rd party / built-ins and relative imports for maintainability. I took this approach in other files too...happy to back this out if undesired! Otherwise we should adopt as one of our codebase style guidelines IMO.